samples/common/mcumgr_bt_ota_dfu/Kconfig

```diff
@@ -111,22 +111,20 @@ config NCS_SAMPLE_MCUMGR_BT_OTA_DFU_SPEEDUP
 	  size. The buffer configuration should be the same in the Network Core
 	  and Application Core image.
 
-if NCS_SAMPLE_MCUMGR_BT_OTA_DFU_SPEEDUP
-
 config BT_L2CAP_TX_MTU
-	default 247 if BT_HCI_HOST
+	default 498 if BT_HCI_HOST && NCS_SAMPLE_MCUMGR_BT_OTA_DFU_SPEEDUP
+	default 247 if BT_HCI_HOST && NCS_SAMPLE_MCUMGR_BT_OTA_DFU
 
 config BT_BUF_ACL_TX_SIZE
 	default 251
 
 config BT_BUF_ACL_RX_SIZE
-	default 251
+	default 502 if NCS_SAMPLE_MCUMGR_BT_OTA_DFU_SPEEDUP
+	default 251 if NCS_SAMPLE_MCUMGR_BT_OTA_DFU
 
 config BT_CTLR_DATA_LENGTH_MAX
 	default 251 if HAS_BT_CTLR
```

Comment:
This one must also depend on `NCS_SAMPLE_MCUMGR_BT_OTA_DFU_SPEEDUP` or `NCS_SAMPLE_MCUMGR_BT_OTA_DFU`